### PR TITLE
EWPP-2718: Update ECL to 3.5.0 version.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by imanoleguskiza — DISMISSED** [review]

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
We need to rename the patch files.

**Review by 22Alexandra — DISMISSED** [review]

**Review by 22Alexandra — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "behat/behat": "^3.10",
         "behat/mink-extension": "^2.3.1",
         "composer/installers": "~1.5",
+        "consolidation/annotated-command": "4.5.6",
```
Comment (upchuk) [inline composer.json]: Is this still needed?

File: tests/src/FunctionalJavascript/JavascriptBehavioursTest.php [hunk 2]
```diff
@@ -123,7 +123,7 @@ public function testEclMultiSelect(): void {
 
     // Assert all options are visible.
     $options = [
-      'Select all',
+      'Select all (4)',
```
Comment (upchuk) [inline tests/src/FunctionalJavascript/JavascriptBehavioursTest.php:126]: Why is this happening? We cannot just change the test cause drone fails. There is a reason we have tests and that is to check if we break stuff. Not that if we break stuff, we just adapt the test. So please confirm what this is.

**Review by upchuk — CHANGES_REQUESTED** [review]
Please

**Review by upchuk — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**